docs/user-guide/nginx-configuration/annotations.md

```diff
@@ -28,7 +28,8 @@ You can add these Kubernetes annotations to specific Ingress objects to customiz
 |[nginx.ingress.kubernetes.io/auth-tls-verify-client](#client-certificate-authentication)|string|
 |[nginx.ingress.kubernetes.io/auth-tls-error-page](#client-certificate-authentication)|string|
 |[nginx.ingress.kubernetes.io/auth-tls-pass-certificate-to-upstream](#client-certificate-authentication)|"true" or "false"|
-|[nginx.ingress.kubernetes.io/auth-tls-match-cn](#client-certificate-authentication)|string|
```

Comment:
let's not deprecate an annotation. Instead, the old behavior should be called this way, and the new one can have a new name

Reply:
makes sense! I will keep the original one here as the regex version, and then only add a new one for the strict mode!